data_sources.py

```diff
@@ -8,7 +8,11 @@
                  ],
                  "Liberecky": [
                               "U2HRAD01_T_N.csv.zip",
-			      "U2STRR01_T_N.csv.zip",
+			                        "U2STRR01_T_N.csv.zip",
```

Comment:
Díky za commit. 
Je to téměř OK.
V rámci úprav došlo k přidání mezer na řádku s libereckym zdrojem "U2STRR01_T_N.csv.zip",
To by chtělo upravit, aby tam ta změna nebyla.
Pak udělat add / commit / push 
a můžeme to dát do hlavní větve.